src/lib/markbind/src/parser.js

```diff
@@ -27,107 +26,16 @@ cheerio.prototype.options.decodeEntities = false; // Don't escape HTML entities
 const {
   ATTRIB_INCLUDE_PATH,
   ATTRIB_CWF,
-  IMPORTED_VARIABLE_PREFIX,
 } = require('./constants');
 
 class Parser {
-  constructor(options) {
-    this._options = options || {};
-    this._fileCache = {};
+  constructor(config) {
+    this.variablePreprocessor = config.variablePreprocessor;
     this.dynamicIncludeSrc = [];
     this.staticIncludeSrc = [];
     this.missingIncludeSrc = [];
   }
 
-  /**
```

Comment:
Most of the methods relating to page variable extraction and rendering are completely changed (there's a high level overview of the changes in the original post).
Do let me know if you prefer me to point out the characteristics (as per the overview) in the original code!